pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -174,21 +178,26 @@ stdenv.mkDerivation rec {
     EventKit
     GSS
     MetalKit
-  ] ++ lib.optional libGLSupported libGL;
+  ] ++ lib.optionals libGLSupported [
+    libGL
+  ] ++ lib.optionals stdenv.hostPlatform.isMinGW [
+    vulkan-headers
+    vulkan-loader
+  ];
```

Comment:
Why only for mingw? Shall we also add vulkan support to linux builds?

Reply:
This isn't a mingw only thing. See https://github.com/NixOS/nixpkgs/blob/b5c956262d950cd2accccbe6a1e7fe768d5efaad/pkgs/development/libraries/qt-6/modules/qtbase.nix#L142-L143. Vulkan appears to be unsupported for Darwin. Did the conditional at the bottom here to avoid triggering a mass rebuild.